frontend/src/scenes/insights/LineGraph.js

```diff
@@ -21,6 +21,16 @@ Chart.defaults.global.elements.line.tension = 0
 
 const noop = () => {}
 
+const commonTooltipsOptions = {
+    enabled: true,
+    yPadding: 10,
+    xPadding: 10,
+    caretPadding: 0,
+    backgroundColor: '#1dc9b7',
+    titleFontColor: '#ffffff',
```

Comment:
Tried to change this to have colors that have strong contrast ratio. Current scheme doesn't work very well, but `titleFontColor` doesn't actually change the color of the label.